docs/src/examples/components/Slot/index.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react'
+
+export default () => <div>No examples</div>
```

Comment:
Maybe we can add here some description of the component and why it was introduced. This is not blocking this PR from being merged however.

Reply:
will do, thx